### ISSUE TITLE
 Buttons do not have an accessible name

### ISSUE THREAD

**Issue by SQLJames** [razonyang/hugo-theme-bootstrap#186]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

Failing Elements
--
```button<button class="btn btn-search disabled position-absolute left-0" type="submit">```

**Comment by SQLJames** [comment]
https://github.com/razonyang/hugo-theme-bootstrap/pull/187

**Comment by razonyang** [comment]
Closed by #187
